models/index.js

```diff
@@ -73,22 +78,23 @@ Idea.hasMany(Comment, {
 });
 
 Idea.belongsToMany(User, {
-  through : IdeaUpvote,
-  as: 'upvoter',
   foreignKey: 'idea_id',
+  through: IdeaUpvote,
+  as: 'upvoter',
 });
 
-// Idea.hasMany(IdeaUpvote, {
-//   foreignKey: 'idea_id',
-// });
+Idea.hasMany(IdeaUpvote, {
+  through: IdeaUpvote,
+  as: 'upvoter',
+  foreignKey: 'idea_id',
+});
```

Comment:
Hey @jmichaelbrown8, in merging changes from the idea upvote branch, we now have an alias conflict over "upvoter", We'll need a new alias name for this.

Reply:
Or maybe I did the merge wrong and the hasMany isn't needed? Not sure in this case without looking through uses more.